documentation/docs/install-pmm/install-pmm-server/deployment-options/docker/backup_container.md

```diff
@@ -21,38 +12,266 @@ To check used Grafana plugins:
+    Use this method if your PMM Server runs with Podman and SystemD:
+
+    **Example deployment via SystemD service**
+    ```sh
+    # Podman with named volume via SystemD
+    systemctl --user status pmm-server
+    ```
 
+    **Create Podman volume backup**
     ```sh
-    mkdir pmm-data-backup && cd pmm-data-backup
+    # Stop PMM Server service
+    systemctl --user stop pmm-server
+
+    # Create backup volume with timestamp
+    BACKUP_VOLUME="pmm-data-backup-$(date +%Y%m%d-%H%M%S)"
+    podman volume create $BACKUP_VOLUME
+
+    # Copy data from current volume to backup volume
+    podman run --rm -v pmm-server:/from -v $BACKUP_VOLUME:/to alpine ash -c 'cd /from ; cp -av . /to'
+
+    # Verify backup
+    podman run --rm -v $BACKUP_VOLUME:/backup alpine ls -la /backup
+
+    # Restart PMM Server service
+    systemctl --user start pmm-server
+
+    # Note backup volume name for restoration
+    echo "Backup volume created: $BACKUP_VOLUME"
     ```
 
-4. Back up the data:
+=== "Kubernetes (Helm)"
+    Use this method if your PMM Server runs on Kubernetes via Helm. Requires `StorageClass` and `VolumeSnapshotClass` that support snapshots. Check with your Kubernetes provider for availability.
 
+    **Example deployment**
     ```sh
-    docker cp pmm-server-backup:/srv .
+    helm install pmm percona/pmm
     ```
 
-5. Verify the backup was created successfully:
+    **Create Kubernetes volume snapshot**
     ```sh
-    ls -la srv/
+    # Check available storage classes and snapshot classes
+    kubectl get storageclass
+    kubectl get volumesnapshotclass
+
+    # Create volume snapshot
+    cat <<EOF | kubectl apply -f -
+    apiVersion: snapshot.storage.k8s.io/v1
+    kind: VolumeSnapshot
+    metadata:
+      name: pmm-backup-$(date +%Y%m%d-%H%M%S)
+      labels:
+        app.kubernetes.io/name: pmm
+    spec:
+      source:
+        persistentVolumeClaimName: pmm-storage-pmm-0
+      volumeSnapshotClassName: your-snapshot-class
+    EOF
+
+    # Verify snapshot creation
+    kubectl get volumesnapshot -l app.kubernetes.io/name=pmm
+    ```
+
+=== "Universal container copy"
+
+    While this method works universally, the deployment-specific methods above are more efficient and preserve storage structures better. This method works for all deployment types as a fallback option when you're unsure about your deployment type or you need a quick backup without determining volume setup.
+    {.power-number}
+
+    1. Stop the running PMM Server container:
+
+        ```sh
+        docker stop pmm-server
+        ```
+
+    2. Rename the container to preserve it as a backup source:
+
+        ```sh
+        docker rename pmm-server pmm-server-backup
+        ```
+
+    3. Create a backup subdirectory and navigate to it:
+
+        ```sh
+        mkdir pmm-data-backup-$(date +%Y%m%d-%H%M%S) && cd pmm-data-backup-$(date +%Y%m%d-%H%M%S)
+        ```
+
+    4. Back up the data:
+
+        ```sh
+        docker cp pmm-server-backup:/srv .
```

Comment:
```suggestion
        docker cp pmm-server-backup:/srv/. .
```
Otherwise it will create an "srv" directory inside of `pmm-data-backup-...`, which is redundant.